src/MultiFormats/Cryptography/Keccak.cs

```diff
@@ -0,0 +1,129 @@
+﻿/*
+ * The package was named SHA3 it is really Keccak
+ * https://medium.com/@ConsenSys/are-you-really-using-sha-3-or-old-code-c5df31ad2b0
+ * See 
+ * 
+ * The SHA3 package doesn't create .Net Standard package.
+ * This is a copy of https://bitbucket.org/jdluzen/sha3/raw/d1fd55dc225d18a7fb61515b62d3c8f164d2e788/SHA3/SHA3.cs
+ */
+
+using System;
+
+namespace MultiFormats.Cryptography
+{
+    internal abstract class Keccak : System.Security.Cryptography.HashAlgorithm
+    {
+        #region Implementation
+
+        public const int KeccakB = 1600;
+        public const int KeccakNumberOfRounds = 24;
+        public const int KeccakLaneSizeInBits = 8 * 8;
+
+        public readonly ulong[] RoundConstants;
+
+        protected ulong[] state;
+        protected byte[] buffer;
+        protected int buffLength;
+
+        protected int keccakR;
+
+        public int KeccakR { get => keccakR; protected set => keccakR = value; }
+
+        public int SizeInBytes => KeccakR / 8;
+
+        public int HashByteLength => HashSizeValue / 8;
+
+        public override bool CanReuseTransform => true;
+
+        protected Keccak(int hashBitLength)
+        {
+            if (hashBitLength != 224 && hashBitLength != 256 && hashBitLength != 384 && hashBitLength != 512)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)